src/Model/Svg/Transformations.php

```diff
@@ -31,7 +31,7 @@ class Transformations
      * @see self::mapFromAttribute()
      * @param string $name Parameter name (e.g. color, bold)
      * @param string $value Parameter value (e.g. black, yes)
-     * @return string[] Numerical array, [0] = attribute name, [1] = value
+     * @return string[]|false[] Numerical array, [0] = attribute name, [1] = value
      */
```

Comment:
I'm nitpicking, but shouldn't this be `bool`, i.e. a type and not a value?

Reply:
This is a practice widely used in MW to denote the only boolean value the function can return is false. Since it's also accepted by every linter we're using, I don't see a problem with it.

Reply:
Yeah I've seen it either with `false` outright or `bool` and then documented as 'always false'. I think `false` is fine.

My bigger question is -- is this an array of everything-false, or is this false, and if it's an array of all-false, then why do we need it at all?

Reply:
This question isn't a blocker for merging, so I'm merging it in, but I will make the point that this documentation is a little confusing in general, and we might want to take a look at how to better document this method in general.